sdk/common/configuration.go

```diff
@@ -80,7 +78,7 @@ func (configuration *NSConfiguration) CompleteNSConfiguration() {
 		configuration.OutgoingNscLabels = getEnv(outgoingNscLabelsEnv, "Outgoing labels", false)
 	}
 
-	configuration.TracerEnabled, _ = strconv.ParseBool(getEnv(tracerEnabled, "Tracer enabled", false))
+	configuration.TracerEnabled = tools.IsOpentracingEnabled()
```

Comment:
@denis-tingajkin I think we do not need this property into configuration?

Reply:
thanks, fixed